201-cdn-oms-solution/azuredeploy.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "workspaceRegionId": {
+      "value": "eastus"
+    },
+    "pricingTier": {
+      "value": "Free"
+    },
+    "solutionName": {
+      "value": "TestSolution"
+    },
+    "workspaceName": {
+      "value": "testomsomsoms"
```

Comment:
use GEN-UNIQUE-20 etc as resource name, to ensure name uniqueness while the automated test/validation kicks in.